test/sockets/test_getaddrinfo.c

```diff
@@ -187,9 +238,10 @@ int main() {
   memset(&hints, 0, sizeof(hints));
   hints.ai_family = AF_INET6;
   hints.ai_socktype = SOCK_STREAM;
-  err = getaddrinfo("www.mozilla.org", "90", &hints, &servinfo);
+  err = getaddrinfo("www.google.com", "90", &hints, &servinfo);
```

Comment:
I changed the address here since www.mozilla.org doesn't have an ipv6 address.

Reply:
Huh... that is surprising. But I confirmed and it seems to be the case...